features/rules/ALB/ALB031_Alignment-layouts-default-case.feature

```diff
@@ -0,0 +1,19 @@
+@implementer-agreement
+@ALB
+@version1
+Feature: ALB031 - Alignment Layouts Default Case
+The rule verifies that alignments are nested in accordance with Concept Template 4.1.4.4.1.1
+which represents the default case for nesting alignment layouts.
+Alignment layouts are abbreviated as follows: horiz = IfcAlignmentHorizontal, vert = IfcAlignmentVertical,
+and cant = IfcAlignmentCant
+
+  Scenario: Agreement on IfcAlignment layout nesting per 4.1.4.4.1.1.
+
+      Given A model with Schema 'IFC4.3'
+      Given an .IfcAlignment.
+      Given a relationship .IfcRelAggregates. ^does not exist^ from .IfcAlignment. to .IfcAlignment. and following that
```

Comment:
I'm a bit confused by the combination of `^does not exist^` and `and following that`.  If it doesn't exist you can also not follow (traverse) along that relationship edge.

```suggestion
      Given a relationship .IfcRelAggregates. ^does not exist^ from .IfcAlignment. to .IfcAlignment.
```